### PR TITLE
feat: migrate to github actions

### PR DESCRIPTION
**Pull request by yohanelly95**

### CODE REVIEW

File: scripts/gasCompare.js [hunk 1]
```diff
@@ -79,21 +76,15 @@ let gasCompare = async () => {
...
-if(gasChangeData.length!==0){
-    
-        await commenter.createOrUpdateComment('gasCompare', markdownstring ).catch(err=>{  
-                console.log(markdownstring);
-        })
+
+    let markdownString = markdown(gasChangeData, coloumn);
+
+    if (gasChangeData.length !== 0) {
+        // Write the markdown to a file called gasCompareOutput.json
```
Comment (SkandaBhat) [inline scripts/gasCompare.js:83]: Write the gas compare output to the console as well. This way we can at least see the gas compare in the actions logs.

**Review by SkandaBhat — CHANGES_REQUESTED** [review]
Please address the comments.

File: scripts/gasCompare.js (hunk 1 above)
Comment (yohanelly95) [reply scripts/gasCompare.js:83]: Okay, will note this on the gas compare issue and tackle it then. Works ? yohan/raz-416-gascompare-needs-to-be-added-to-github-actions @SkandaBhat

Comment (yohanelly95) [reply scripts/gasCompare.js:83]: can you confirm @SkandaBhat, since we will not be running gasCompare on the CI temporarily.

**Comment by yohanelly95** [comment]
@SkandaBhat requires stop building on circle ci

**Review by SamAg19 — APPROVED** [review]

**Review by SkandaBhat — APPROVED** [review]
lgtm

**Pull request merged**